pkg/helper/envVar.go

```diff
@@ -0,0 +1,30 @@
+package helper
+
+import (
+	"k8s.io/kubernetes/pkg/api"
+)
+
+//CacheEnvVars ...
```

Comment:
Good catch. I meant to actually describe it. Was just tired of the linter yelling at me. I'll fix and then merge.